app/views/profiles/edit.html.erb

```diff
@@ -1,11 +1,29 @@
 <h1 class="text-2xl text-center">Edit Profile</h1>
 
-<div class="max-w-prose px-2">
-  <%= form_with model: @profile do |form| %>
+<div class="max-w-prose px-2 mx-auto my-4">
+  <%= form_for @profile_form, format: :html do |form| %>
```

Comment:
Does this type of pattern only work with `form_for`? Only asking because `form_for` is [considered soft-deprecated](https://guides.rubyonrails.org/form_helpers.html#using-form-tag-and-form-for)

Reply:
ah, didn't realize! I'll move to `form_with`

Reply:
learn something new every day, didn't either know this lol